packages/mds-agency/api.ts

```diff
@@ -832,6 +832,7 @@ function api(app: express.Express): express.Express {
     const { device_id } = req.params
 
     const { provider_id } = getAuth(req)
+    const name = providerName(provider_id || 'unknown')
```

Comment:
This does't look right. `const name = providerName(provider_id) || 'unknown'` perhaps?